webapp/src/Service/SubmissionService.php

```diff
@@ -173,7 +173,7 @@ public function getSubmissionList(array $contests, array $restrictions, int $lim
             if ($restrictions['verified']) {
                 $queryBuilder->andWhere('j.verified = 1');
             } else {
-                $queryBuilder->andWhere('j.verified = 0 OR (j.verified IS NULL AND s.judgehost IS NULL)');
+                $queryBuilder->andWhere('j.verified = 0 OR (j.verified IS NULL AND j.judgehost IS NULL)');
```

Comment:
@meisterT are we even setting the judgeost on a judging?

Reply:
We right now set the judgehost only when we pop back the compilation result on compile error. We should probably change that so that we remove that field from the table, but it's independent from this PR.

For this PR I would just remove the `AND ...` part.